datacenter/ucp/2.1/guides/admin/monitor-and-troubleshoot/troubleshoot-configurations.md

```diff
@@ -71,6 +73,44 @@ On failure the command exits with an error code, and no output.
 To learn more about the `etcdctl` utility, check the
 [etcd official documentation](https://coreos.com/etcd/docs/latest/).
 
+## RethinkDB Database
+
+User and organization data for Docker Datacenter is stored in a RetinkDB
+database which is replicated across all conroller nodes in the UCP cluster.
+
+Replication and failover of this database is typically handled automatically by
+UCP's own configuration management processes, but detailed database status and
+manual reconfiguration of database replication is available through a command
+line tool available as part of UCP.
+
+The examples below assume you are logged in with ssh into a UCP controller node.
```

Comment:
UCP controller -> UCP manager
I know we use "controller" in some places, but I'm trying to get rid of it now that manager nodes and UCP controllers are 1:1.